e2e/testcases/correction-birth/correct-birth-record-2.spec.ts

```diff
@@ -0,0 +1,1358 @@
+    expect(page.url().includes('reason')).toBeTruthy()
+
+    await expect(page.getByRole('button', { name: 'Continue' })).toBeDisabled()
+
+    await page
+      .getByLabel('Informant provided incorrect information (Material error)')
+      .check()
+
+    /*
+     * Expected result: should enable the continue button
+     */
+
+    await page.getByRole('button', { name: 'Continue' }).click()
+  })
+
+  test('2.7 Correction summary', async () => {
+    /*
+     * Expected result: should
+     * - navigate to correction summary
+     * - Send for approval button is disabled
+     */
+    expect(page.url().includes('summary')).toBeTruthy()
+    expect(page.url().includes('correction')).toBeTruthy()
+
+    await expect(
+      page.getByRole('button', { name: 'Send for approval' })
+    ).toBeDisabled()
+
+    /*
+     * Expected result: should show
+     * - Original vs correction
+     * - Requested by
+     * - ID check
+     * - Reason for request
+     * - Comments
+     */
+
+    await expect(
+      page.getByText(
+        'Place of delivery (Child)' +
+          'Health Institution' +
+          'Farajaland' +
+          'Central' +
+          'Ibombo' +
+          '-' +
+          '-' +
+          '-' +
+          '-' +
+          '-' +
+          '-' +
+          'Residential address' +
+          'Farajaland' +
+          updatedChildDetails.birthLocation.province +
+          updatedChildDetails.birthLocation.district +
+          updatedChildDetails.birthLocation.town +
+          updatedChildDetails.birthLocation.residentialArea +
+          updatedChildDetails.birthLocation.street +
+          updatedChildDetails.birthLocation.number +
+          updatedChildDetails.birthLocation.zipCode
+      )
+    ).toBeVisible()
+
+    await expect(
+      page.getByText(
+        'Relationship to child (Informant)' +
+          declaration.informant.relationship +
+          updatedInformantDetails.relationship
+      )
+    ).toBeVisible()
+
+    await expect(
+      page.getByText(
+        'Full name (informant)' +
+          formatName(declaration.informant.name[0]) +
+          formatName(updatedInformantDetails)
+      )
+    ).toBeVisible()
+
+    await expect(
+      page.getByText(
+        'Date of birth (informant)' +
+          formatDateTo_ddMMMMyyyy(declaration.informant.birthDate) +
+          formatDateTo_ddMMMMyyyy(updatedInformantDetails.birthDate)
+      )
+    ).toBeVisible()
+
+    await expect(
+      page.getByText(
+        'Nationality (Informant)Farajaland' +
+          updatedInformantDetails.nationality
+      )
+    ).toBeVisible()
+
+    await expect(
+      page.getByText(
+        'Type of ID (Informant)National ID' + updatedInformantDetails.idType
+      )
+    ).toBeVisible()
+    await expect(
+      page.getByText('ID Number (Informant)-' + updatedInformantDetails.id)
+    ).toBeVisible()
+
+    await expect(
+      page.getByText(
+        'Usual place of residence (Informant)' +
```

Comment:
Might be a matter of taste but `joinValuesWith([...array, '')` could achieve the same as the plus signs